js/graviex.js

```diff
@@ -0,0 +1,1039 @@
+                'public': {
+                    'get': [
+                        'markets',
+                        'tickers',
+                        'depth', // Requires market XXXYYY. Optional: asks_limit, bids_limit.
+                        'trades', // Requires market XXXYYY. Optional: limit, timestamp, from, to, order_by
+                        'trades_simple', // Requires market XXXYYY.
+                        'k', // Requires market XXXYYY. Optional: limit, timestamp, period
+                        'k_with_pending_trades', // API endpoints exists almost same as k endpoint skipping for now.
+                        'tickers/{market}', // Get ticker of specific market
+                    ],
+                },
+                'private': {
+                    'get': [
+                        'order_book', // Requires access_key, tonce, signature and market XXXYYY.
+                        'members/me', // Requires access_key, tonce, signature and api secret.
+                        'deposits', // Requires access_key, tonce, signature and api secret. Optional currency, limit and state.
+                        'deposit', // Requires access_key, tonce, signature, txid and api secret.
+                        'deposit_address', // Requires access_key, tonce, signature, currency and api secret.
+                        'orders', // Requires access_key, tonce, signature and api secret. Optional market, state, limit, page, order_by
+                        'order', // Requires access_key, tonce, signature and api secret AND id (unique orderid).
+                        'trades/my', // Requires market XXXYYY, access_key, tonce, signature and api secret . Optional: limit, timestamp, from, to, order_by
+                        'trades/history', // Requires market XXXYYY, access_key, tonce, signature and api secret . Optional: limit, timestamp, from, to, order_by
+                        'orders/history', // Requires market XXXYYY, access_key, tonce, signature and api secret . Optional: limit, timestamp, from, to, order_by
+                    ],
+                    'post': [
+                        'orders', // Requires access_key, tonce, signature and api secret AND market, side(sell/buy), volume. Optional price ord_type
+                        'orders/multi', // Requires access_key, tonce, signature and api secret AND market, orders, orders[side(sell/buy)], orders[volume]. Optional orders[price], orders[ord_type]. To create multiple orders at once
+                        'orders/clear', // Requires access_key, tonce, signature and api secret. Optional side(sell/buy) to only clear orders on one side.
+                        'order/delete', // Requires access_key, tonce, signature and api secret AND id (unique orderid).
```

Comment:
We are missing a lot of endpoints from the Gravies API: https://graviex.net/documents/api_v3
We have to list them all out even if we don't use them in the unified methods: 

- https://github.com/ccxt/ccxt/wiki/Manual#implicit-api
- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#new-exchange-integrations

Reply:
@kroitor, Added those missing endpoints and put up the comment // Not Using Currently In Unified Methods. For endpoints that are not included in UNIFIED Methods. Please check-in latest commit